src/tasks/delete-item-tag-task.ts

```diff
@@ -1,16 +1,14 @@
-// global
-import { Actor, DatabaseTransactionHandler } from 'graasp';
-// other services
-import { ItemService, ItemMembershipService } from 'graasp';
-// local
+import { Actor, DatabaseTransactionHandler, TaskStatus } from '@graasp/sdk';
+import { ItemMembershipService, ItemService } from '@graasp/sdk';
```

Comment:
Combine into single import?
```suggestion
import { Actor, DatabaseTransactionHandler, ItemMembershipService, ItemService, TaskStatus } from '@graasp/sdk';
```